actions/shell/action.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Shell Action'
+description: 'Invoke wake inside the environment-blockci-sifive container'
+inputs:
+  command:
+    description: |
+      The command which will be executed by bash inside the workspace.
+    required: true
+  workspace:
+    description: |
+      The location (relative to the build directory) of the root of the wit
+      workspace.  This should typically match wherever wit/actions/init ran.
+    required: false
+    default: .
+  version:
+    description: |
+      The version of the environment-blockci-sifive docker image to use.  By
+      default, the same version is used as the release tag of the action.
+    required: false
+    default: 0.7.0
```

Comment:
I guess we should update this number, and we probably need to add this to the [steps for making a new release](https://github.com/sifive/environment-blockci-sifive/blob/master/docs/how-to/make-a-release.md).

Reply:
I have another commit that changes this. I'm going to do it as a follow-up PR, though, since you want to get in nodejs.